packages/ERTP/src/issuer.js

```diff
@@ -78,7 +90,10 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
         );
         return payment;
       },
-      getCurrentAmount: () => purseLedger.get(purse),
+      getCurrentAmount: () => {
+        assertKnownPurse(purse);
```

Comment:
Does this check make sense? Purse is not an argument here.

Reply:
It doesn't hurt to leave it in, but I think it's not checking anything.

Reply:
I just checked. purses are never deleted from the purseLedger. We don't need assertKnownPurse. Please delete it.

Reply:
Thanks. I got rid of it.